### ISSUE TITLE
Links missing underline and colors in the admin

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/student-loan-planner-theme#281]
<img width="648" alt="Screenshot 2023-10-10 at 6 24 04 PM" src="https://github.com/equalizedigital/student-loan-planner-theme/assets/13153456/1d3f07a5-6b94-4344-b8d3-59647f72d874">

It's impossible to tell where links start and stop. They should have the same WCAG green color and an underline.
